src/components/colorways/index.js

```diff
@@ -4,13 +4,14 @@ import sa from './sa';
 import jtk from './jtk';
 import kat from './kat';
 import mt3 from './mt3';
+import dcs from './dcs';
 import alphaCodes from './alphaCodes';
 import modCodes from './modCodes';
 
 export default {
   alphaCodes,
   modCodes,
-  list: [...dsa, ...sa, ...gmk, ...jtk, ...kat, ...mt3],
+  list: [...dsa, ...sa, ...gmk, ...jtk, ...kat, ...mt3, ...dcs],
```

Comment:
Putting DCS first will keep the list in alphabetical order.

```suggestion
  list: [...dcs, ...dsa, ...sa, ...gmk, ...jtk, ...kat, ...mt3],
```